controllers/controller.articles.js

```diff
@@ -1,10 +1,14 @@
 const { getArticle } = require('../models/model.articles');
 
 exports.getArticleId = (req, res, next) => {
-    const { article_id } = req.query;
-    console.log(article_id)
-    getArticle(article_id).then((article) => {
-      res.status(200).send({ article });
+  const { article_id } = req.params
+    getArticle(article_id)
+    .then((article) => {
+      if (!article) {
+        res.status(404).send({ msg: 'Article not found' })
```

Comment:
Your model should be using a Promise.reject to pass on that error to the catch block of your controller. The way you have it right now works, but isn't the correct implementation of MVC.